packages/shared/src/components/sidebar/ActivitySection.tsx

```diff
@@ -31,6 +31,7 @@ export function ActivitySection({
         ),
         title: 'Bookmarks',
         path: `${process.env.NEXT_PUBLIC_WEBAPP_URL}bookmarks`,
```

Comment:
I think you can actually just set this to be `href` and you don't need `forcedLink`
The forcedLink is more meant for `extension || webapp` where we have pages, but for webapp you want to force a link or something like that.

Reply:
Noticed in preview that it doesn't work, because if you're not logged in, it always returns a button. But I think we probably can remove that since we want anchor for SEO?